com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/AtlasBuilder.java

```diff
@@ -26,15 +26,11 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-import com.dynamo.bob.Builder;
-import com.dynamo.bob.BuilderParams;
-import com.dynamo.bob.CompileExceptionError;
-import com.dynamo.bob.Task;
+import com.dynamo.bob.*;
```

Comment:
I'm not a fan of wildcard imports as it can result in name conflicts. I would prefer if we don't start doing it.

Reply:
I agree. I'm trying to avoid it as well but that's what IDEA does. I'll fix it